mmv1/third_party/terraform/data_sources/data_source_google_container_attached_install_manifest.go

```diff
@@ -0,0 +1,71 @@
+package google
+
+import (
+	"fmt"
+	"time"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+)
+
+func dataSourceGoogleContainerAttachedInstallManifest() *schema.Resource {
+	return &schema.Resource{
+		Read: dataSourceGoogleContainerAttachedInstallManifestRead,
+		Schema: map[string]*schema.Schema{
+			"location": {
+				Type:     schema.TypeString,
+				Required: true,
+			},
+			"parent": {
+				Type:     schema.TypeString,
+				Required: true,
+			},
+			"cluster_id": {
+				Type:     schema.TypeString,
+				Required: true,
+			},
+			"platform_version": {
+				Type:     schema.TypeString,
+				Required: true,
+			},
+			"manifest": {
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+		},
+	}
+}
```

Comment:
Switching from "parent" to `project`, `region` might be a better UX. If region is identical to location this `region` can be omitted.